src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -45,6 +45,7 @@
 ACRES_PER_SQM = 0.000247105
 HECTARES_PER_SQM = 0.0001
 SQKM_PER_SQM = 0.000001
+NO_LAND_COVER = 'No land cover'
```

Comment:
If we're doing conditionals off of this text, better to make it look like a code I think (`no_land_cover`), so as not to be interpreted as a label and casually changed.